app/jobs/resolution_proofing_job.rb

```diff
@@ -19,7 +19,7 @@ def perform(
     encrypted_arguments:,
     trace_id:,
     should_proof_state_id:,
-    double_address_verification: false,
+    ipp_enrollment_in_progress: false,
```

Comment:
renaming a job argument for a background job needs to be done in multiple steps because of the 50-50 state during deploys.

1. add the new name as with a default value to the job
2. add the new name
3. remove the old name

Reply:
@zachmargolis maybe this is a conversation for slack, but I'm unsure why I need to only account for the 50/50 state in the job code? Does the application code get provisioned differently than the jobs do?

Reply:
On the same box, all code will be the same version, so in-memory code doesn't need to worry about the 50-50 state.

Background jobs get written to the database, so during the 50-50 state, they can be picked up by either old or new boxes. So since we don't know which version the box will write the job is and which version box will perform the job is, we need to be more flexible.